docs: Add wizard getting started documentation (#4452)
unoplatform/uno: doc/articles/get-started-wizard.md

# Welcome to the Uno Platform

Congratulations, you've just created a new project using the Uno Platform project templates.

> * You can give a try to the [Single Page app getting started guide](https://platform.uno/docs/articles/getting-started-tutorial-1.html)

Reviewer: ```suggestion
* You can give the [Single Page app getting started guide](https://platform.uno/docs/articles/getting-started-tutorial-1.html) a try
```

* Next you can try our [Bug Tracker sample walk-through](https://platform.uno/docs/articles/getting-started-tutorial-2.html)

> * More advanced examples in our [Uno.Samples repository](https://github.com/unoplatform/uno.samples)

Reviewer 2: ```suggestion
* More advanced examples in our [Uno.Samples repository](https://github.com/unoplatform/uno.samples)
* Fork a fully-fledged [Ch9 application and source code ](https://platform.uno/code-samples/#ch9)
```

* You can find detailed documentation on Uno topics [on our web site](https://platform.uno/docs/articles/intro.html).

## Common Issues
The Uno Platform features and support is constantly evolving, but you may encounter issues while building your application.

### XAML Hot Reload for the UWP head may not work properly in Visual Studio 2019 16.7 and earlier
The [XAML Hot Reload](https://docs.microsoft.com/en-us/visualstudio/xaml-tools/xaml-hot-reload?view=vs-2019) feature [does not work in Visual Studio 2019 16.7](https://developercommunity.visualstudio.com/content/problem/996417/xaml-hot-reload-hot-reload-is-not-working-in-share.html) and earlier for Uno projects. This will be fixed in 16.8 and later.

### The XAML editor shows `The type 'page' does not support direct content` message
XAML Intellisense [is not working properly](https://developercommunity.visualstudio.com/content/problem/587980/xaml-intellisense-does-not-use-contentpropertyattr.html) in Visual Studio when the active project is not the UWP one. 

> To work around this issue, close all XAML editors, open a C# file and selec the UWP in the top left drop-down list of the text editor sector. Once selected, re-open the XAML file.

Reviewer: ```suggestion
To work around this issue, close all XAML editors, open a C# file and select 'UWP' in the top left drop-down list of the text editor sector. Once selected, re-open the XAML file.
```

### The XAML editor may be empty
The XAML editor may be empty when the active editor project is not UWP.

> To work around this issue, close all XAML editors, open a C# file and selec the UWP in the top left drop-down list of the text editor sector. Once selected, re-open the XAML file.

Reviewer: ```suggestion
To work around this issue, close all XAML editors, open a C# file and select 'UWP' in the top left drop-down list of the text editor sector. Once selected, re-open the XAML file.
```

### The path may be too long

> The project may fail to build is the path of the project is too long.

Reviewer: ```suggestion
The project may fail to build if the path of any file in the project is [too long](https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation).
```

> To work around this issue, either :

Reviewer: ```suggestion
To work around this issue, either:
```

> - Move the project to a shorter path

Reviewer: ```suggestion
- Move the project to a shorter root path, or
```

- Enable Long Path support in Windows 10 by executing this:
```
reg ADD HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem /v LongPathsEnabled /t REG_DWORD /d 1
```

### `x:Name` variable is not available in code-behind
Visual Studio 16.7 and earlier versions [do not refresh the intellisense cache ](https://developercommunity.visualstudio.com/content/problem/588021/the-compile-itemgroup-intellisense-cache-is-not-re.html)properly, causing variables to be incorrectly defined.

To fix this issue, build your project once, close the solution and reopen it.

### "Missing value for TargetPlatformWinMDLocation property" when adding a project reference
This issue is caused by [VS 2019 support for SDK-Style projects](https://developercommunity.visualstudio.com/content/problem/1170010/missing-value-for-targetplatformwinmdlocation-prop.html).

> To add a reference change the list 

Reviewer: Incomplete?

> ### Build fails `error : Error reading response`

Reviewer: ```suggestion
### Build fails with `error : Error reading response`
```

In general, this error happens when the XAML parser detects a syntax error. Fixing the error generally fixes the build.

This error may happen occasionally without any explicit error message, rebuilding the project may fix the issue.

> ### Event cannot be added

Reviewer: ```suggestion
### Event handler cannot be added automatically
```

> Events [cannot be automatically](https://github.com/unoplatform/uno/issues/1348#issuecomment-520300471) added using the XAML editor. 

Reviewer: ```suggestion
Event handlers [cannot be automatically](https://github.com/unoplatform/uno/issues/1348#issuecomment-520300471) added using the XAML editor. 
```

> A workaround is to use the [`x:Bind` to events feature](https://github.com/unoplatform/uno/blob/master/doc/articles/features/windows-ui-xaml-xbind.md#examples) and manually create a method in the code-behind.

Reviewer: maybe explain the advantage of this approach (that it doesn't require remembering a complicated method signature?)
